scripts/install.js

```diff
@@ -56,6 +57,8 @@ function download(url, dest, cb) {
     }
   };
 
+  console.log('Start downloading binary at', url);
```

Comment:
Yes it was. The user wont get any progress output until we get a response which won't happen for things like connection timeouts. 

Tracker groups not appear to produce any output. I could be wrong, the docs aren't useful.